refund.sol

```diff
@@ -17,96 +17,22 @@ along with the DAO.  If not, see <http://www.gnu.org/licenses/>.
-        whiteList[0xe4ae1efdfc53b73893af49113d8694a057b9c0d1] = true;
-        whiteList[0x0737a6b837f97f46ebade41b9bc3e1c509c85c53] = true;
-        
-        whiteList[0x52c5317c848ba20c7504cb2c8052abd1fde29d03] = true;
-        whiteList[0x5d2b2e6fcbe3b11d26b525e085ff818dae332479] = true;
-        whiteList[0x057b56736d32b86616a10f619859c6cd6f59092a] = true;
-    //    whiteList[0x304a554a310c7e546dfe434669c62820b7d83490] = true;
-        whiteList[0x4deb0033bb26bc534b197e61d19e0733e5679784] = true;
-        whiteList[0x35a051a0010aba705c9008d7a7eff6fb88f6ea7b] = true;
-        whiteList[0x9da397b9e80755301a3b32173283a91c0ef6c87e] = true;
-        whiteList[0x0101f3be8ebb4bbd39a2e3b9a3639d4259832fd9] = true;
-        whiteList[0xbcf899e6c7d9d5a215ab1e3444c86806fa854c76] = true;
-        whiteList[0xa2f1ccba9395d7fcb155bba8bc92db9bafaeade7] = true;
-        
-        whiteList[0xd164b088bd9108b60d0ca3751da4bceb207b0782] = true;
-        whiteList[0x1cba23d343a983e9b5cfd19496b9a9701ada385f] = true;
-        whiteList[0x9fcd2deaff372a39cc679d5c5e4de7bafb0b1339] = true;
-        whiteList[0x0e0da70933f4c7849fc0d203f5d1d43b9ae4532d] = true;
-        whiteList[0xbc07118b9ac290e4622f5e77a0853539789effbe] = true;
-        whiteList[0xacd87e28b0c9d1254e868b81cba4cc20d9a32225] = true;
-        whiteList[0x5524c55fb03cf21f549444ccbecb664d0acad706] = true;
-    //    whiteList[0xfe24cdd8648121a43a7c86d289be4dd2951ed49f] = true;
-        whiteList[0x253488078a4edf4d6f42f113d1e62836a942cf1a] = true;
-    //    whiteList[0xb136707642a4ea12fb4bae820f03d2562ebff487] = true;
-        
-        whiteList[0xf14c14075d6c4ed84b86798af0956deef67365b5] = true;
-    //    whiteList[0xaeeb8ff27288bdabc0fa5ebb731b6f409507516c] = true;
-        whiteList[0x6d87578288b6cb5549d5076a207456a1f6a63dc0] = true;
-        whiteList[0xaccc230e8a6e5be9160b8cdf2864dd2a001c28b6] = true;
-    //    whiteList[0x4613f3bca5c44ea06337a9e439fbc6d42e501d0a] = true;
-    //    whiteList[0x84ef4b2357079cd7a7c69fd7a37cd0609a679106] = true;
-    //    whiteList[0xf4c64518ea10f995918a454158c6b61407ea345c] = true;
-    }
-
-    function withdrawFromChildDAO(DAO _child) {
-        // to be replaced by a time which allows the direct withdraw to be finished before the childDAO withdraw starts
-        if (now < fixChildDAOsListTime + 4 weeks) throw;
-        if (!whiteList[_child]
-            || _child.lastTimeMinQuorumMet() > fixChildDAOsListTime
-            || _child.privateCreation() != mother)
-            throw;
-
-        withdraw(_child);
-    }
-
-    function withdraw(){
-        withdraw(mother);
-    }
+    function withdraw(DAO _dao) {
+        if (block.number < 0xfffff) throw; // to be replaced by HF block number
+        if (!refundChild) {
+            refundChild = true;
+            childRefund = new ChildRefund()
+            if (!childRefund.send(174558484783966581114679) || address(childRefund) == 0) throw;
```

Comment:
yes, basically all ETH that is in all childDAO, minus the attacker (white and black) daos.